test/realtime_server_goal_handle_tests.cpp

```diff
@@ -109,182 +125,197 @@ struct FeedbackCallback
   }
 };
 
-std::shared_ptr<TwoIntsActionClient>
+std::shared_ptr<ClientGoalHandle>
 send_goal(
+  rclcpp::Node::SharedPtr node,
+  std::shared_ptr<rclcpp_action::Client<Fibonacci>> ac,
   const std::string & server_name,
-  FeedbackCallback * cb = nullptr)
+  ActionClientCallbacks & client_callbacks)
```

Comment:
Is this general ROS2 convention to avoid taking a stance on where the `&` should go? ;) 
I personally prefer to appended to the typename. Feel free to ignore.

Reply:
:+1: thank you